src/main/scripts/org/reaktivity/specification/nukleus/oauth/control/resolve/with.roles/controller.rpt

```diff
@@ -30,9 +30,10 @@ write 0x00000011
 write ${controlR05C.nextCorrelationId()}
 write ${core:string("oauth")}
 write [0x05] "RS256"
-write 12
-write [0x05] "role1"
-write [0x05] "role2"
+write 6
+write [0x01] "1"
+write [0x01] "2"
+write [0x01] "3"
```

Comment:
Please change this back to `role1`, `role2` and add `role3` if needed, it will be much more readable after you do that.